src/cdk/keycodes/keycodes.md

```diff
@@ -0,0 +1,26 @@
+### KeyCodes
+ 
+Commonly used keycode constants.
+
+#### Example
+```ts
+import {Directive} from '@angular/core';
+import {UP_ARROW, DOWN_ARROW, LEFT_ARROW, RIGHT_ARROW} from '@angular/cdk/keycodes';
+
+@Directive({
+  selector: '[count-arrows]'
+  host: {
+    (keypress): 'handleKeyPress($event)'
+  }
+})
+export class ArrowCounterDirective {
+  arrowPressCount = 0;
+
+  handleKeyPress(event: KeyboardEvent) {
+    if (event.codeCode in [UP_ARROW, DOWN_ARROW, LEFT_ARROW, RIGHT_ARROW]) {
```

Comment:
1. `event.codeCode` -> `event.keyCode`.
2.  This check will look for a property that corresponds to `event.keyCode` in the `Array` prototype. The correct way to check it is `[UP_ARROW, DOWN_ARROW, LEFT_ARROW, RIGHT_ARROW].includes(event.keyCode)`.